Xamarin.Essentials/Vibration/Vibration.uwp.cs

```diff
@@ -1,20 +1,55 @@
 ﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
 using Windows.Foundation.Metadata;
+using Windows.Gaming.Input;
 using Windows.Phone.Devices.Notification;
 
 namespace Xamarin.Essentials
 {
     public static partial class Vibration
     {
+        static Gamepad firstGamepad;
+
+        static Gamepad FirstGamepad
+        {
+            get
+            {
+                if (firstGamepad == null)
+                    firstGamepad = Gamepad.Gamepads.FirstOrDefault();
+
+                return firstGamepad;
+            }
+        }
+
         internal static bool IsSupported
-            => ApiInformation.IsTypePresent("Windows.Phone.Devices.Notification.VibrationDevice") && DefaultDevice != null;
+            => FirstGamepad != null || (ApiInformation.IsTypePresent("Windows.Phone.Devices.Notification.VibrationDevice") && DefaultDevice != null);
 
         static VibrationDevice DefaultDevice => VibrationDevice.GetDefault();
 
-        static void PlatformVibrate(TimeSpan duration) =>
-            DefaultDevice.Vibrate(duration);
+        static void PlatformVibrate(TimeSpan duration)
+        {
+            if (FirstGamepad != null)
+            {
+                FirstGamepad.Vibration = new GamepadVibration { RightMotor = 0.85 };
+                Task.Delay(duration).ContinueWith((t) => FirstGamepad.Vibration = new GamepadVibration { });
```

Comment:
Use the default struct constructor here. Also, possibly to indicate that we are not interested in the previous task we could discard it with _ The second is just personal preference though :)